logic/src/commonMain/kotlin/com/wire/kalium/logic/data/conversation/ConversationRepository.kt

```diff
@@ -292,30 +291,26 @@ internal class ConversationDataSource internal constructor(
     }
 
     override suspend fun observeConversationListDetails(): Flow<List<ConversationDetails>> =
-        conversationDAO.getAllConversationDetails().map { it.map(conversationMapper::fromDaoModelToDetails) }
-
+        conversationDAO.getAllConversationDetails()
+//             .debounce(1.seconds) TODO check performance
```

Comment:
do we need these commented lines anymore?

Reply:
For now, no